tools/export/iar/iar_definitions.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
Thanks for adding it to the description, but can it be added to code in comments and/or `docs` folder as well? I'm trying to think of people who are adding support for this and it's not immediately clear to me of the correlation.